src/main/java/com/khutircraftubackend/auth/ReceiveAdvertisingEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.khutircraftubackend.auth;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Table (name = "receive_advertising")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ReceiveAdvertisingEntity {
```

Comment:
this could be `MailingListEntity` or `CampaignEntity`
in case of `Campaign` it might be not only email but some discount, promotion code, referral, etc., so if we decide naming it `Campaign` we'd need to think about its Architecture in advance.